scilpy/io/utils.py

```diff
@@ -206,22 +206,22 @@ def add_sh_basis_args(parser, mandatory=False):
                         help=help_msg)
 
 
-def validate_nbr_processes(parser, args, default_nbr_cpu=None):
+def validate_nbr_processes(parser, args):
     """ Check if the passed number of processes arg is valid.
-    If not valid (0 < nbr_cpu_to_use <= cpu_count), raise parser.error.
+    If not valid (0 < args.nbr_processes <= cpu_count), raise parser.error. If
```

Comment:
Should we not replace `<` by `<=`, since `args.nbr_processes = 0` does not raise a parser.error ?

Reply:
Not sure if I get it:  as you say, `args.nbr_processes = 0` does NOT raise any error, and thus it is a valid value, isn't it?

Reply:
Yes, but the range in parentheses seems to refers to valid values, and as it as right now 0 is excluded from it.

Reply:
OK, I see. Looks like the wording could be improved: I grant that the condition within the brackets is not right if mean the valid values, but reading "If not valid", I'd assume that the values within the brackets refer to not valid/invalid values, which is not the case. Re-worked the thing in e733d7d. Also realized that the `nbr_cpu == 0` will never happen.

Reply:
Resolving following https://github.com/scilus/scilpy/pull/530#pullrequestreview-833433073.

Thanks for the suggestions @CHrlS98.